samcli/commands/check/resources/lambda_function_pricing.py

```diff
@@ -0,0 +1,144 @@
+"""
+Object to contain Lambda function pricing info.
+"""
+from typing import List, Tuple
+import click
+
+from samcli.commands.check.lib.ask_question import ask
+from samcli.commands.check.resources.unique_pricing_info import UniquePricingInfo
+
+
+class LambdaFunctionPricing(UniquePricingInfo):
```

Comment:
The reason I suggested to have a generic interface (`UniquePricingInfo`) and a special implementation (`LambdaFunctionPricing`) is because you can put lambda specific questions and answers in the implementation where the generic interface can have a generic method which won't expose resource specific stuff to the outside.

Since you created this `UniquePricingInfo`, you can now have an abstract method in that interface;

```
class UniquePricingInfo(ABC):
  
  @abstractmethod
  def ask_questions(self):
    pass
```

And in the `LambdaFunctionPricing` implementation you can implement that method as;

```
class LambdaFunctionPricing(UniquePricingInfo):

  def ask_questions(self):
    # lambda resource specific questions
    self.ask_lambda_function_questions()
    ...
    ...
```

Reply:
I understand. I will implement pricing (and also other features) this way.